lib/smack/Regions.cpp

```diff
@@ -332,14 +309,29 @@ void Regions::visitAtomicRMWInst(AtomicRMWInst &I) {
   idx(I.getPointerOperand());
 }
 
-void Regions::visitMemIntrinsic(MemIntrinsic &I) {
+void Regions::visitMemSetInst(MemSetInst &I) {
+  unsigned length;
+
+  if (auto CI = dyn_cast<ConstantInt>(I.getLength()))
+    length = CI->getZExtValue();
+  else
+    length = std::numeric_limits<unsigned>::max();
+
+  idx(I.getDest(), length);
+}
+
+// Shaobo: we need to visit the source location otherwise
```

Comment:
Put this comment next to the line where you are doing this. Also, maybe remove Shaobo from the comment.

Reply:
Done.